website/admin.py

```diff
@@ -173,58 +174,48 @@ def get_admin_achievements_page(self, user):
             page_title=gettext("title_admin"),
         )
 
-    @route("/mark-as-teacher", methods=["POST"])
-    def mark_as_teacher(self):
+    @route("/mark-as-teacher/<username_teacher>", methods=["GET"])
```

Comment:
I think the problem it's happening because these calls were switched from GET to POST. And, I would prefer these calls to be POST, because we are not getting any data from the server, just modifying, which is exactly what post calls are meant for.

Reply:
I've changed it back to post!